tests/BuildScriptGenerator.Tests/Node/NodePlatformTest.cs

```diff
@@ -125,6 +125,42 @@ public void GeneratedBuildSnippet_HasNpmRunBuildAzureCommand()
             Assert.Contains("npm run build:azure", buildScriptSnippet.BashBuildScriptSnippet);
         }
 
+        [Fact]
+        public void GeneratedBuildSnippet_HasLernaRunBuildCommand_IfLernaJsonFileExists()
```

Comment:
add couple of more tests .. 
1. lerna and lage are both there and we are throwing exception
2. lerna and customrunbuild command is there, customrunbuild should be executed
3. lernarunbuild, customrunbuild and lage command all are there, we should throw exception
4. npmrunbuild and lernarunbuild are there and we should be running lernarunbuild

Reply:
Sure, sounds good